cosmos/event/organizer.py

```diff
@@ -0,0 +1,18 @@
+from cosmos.event.base import PretixService
+
+
+class Organizer(PretixService):
+    """
+    HTTP request service for Pretix Organizers
+
+    Reference: https://docs.pretix.eu/en/latest/api/resources/organizers.html
+    """
+
+    @classmethod
+    def _get_request_url(cls, **kwargs):
+        organizer = kwargs.get("organizer", "")
+        return f"api/v1/organizers/{organizer}"
+
+
+if __name__ == "__main__":
+    print(Organizer.get_all())
```

Comment:
change these into tests, also for the other files

Reply:
I plan to make the tests alongside the Board and Committee models. That way, it is more representative of the real application. If you would like, I can remove these `main()` functions until this is done.

Reply:
yeah please do that